.github/workflows/Agent-Contribution.yaml

```
@@ -0,0 +1,31 @@
+name: Agent Contribution
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  process-agent-contribution:
+    if: contains(github.event.comment.body, '/agent-contribute')
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)